test/call/undefined.js

```diff
@@ -1,11 +1,11 @@
 const assert = require("assert");
-const bootstrap = require("../helpers/bootstrap");
+const bootstrap = require("../helpers/contract/bootstrap");
 
 describe("Undefined", () => {
   describe("Calls", () => {
     const mainContract = "Call";
     const contractFilenames = [];
-    const contractPath = "../contracts/call/";
+    const contractPath = "../../contracts/call/";
```

Comment:
separation of concerns: if we're going to use a directory structure pattern for storing contracts everywhere along with `bootstrap` & co, should tests need to be concerned with path resolution issues, or should it be the job of bootstrap, etc to map out the storage/path pattern for the test?

Reply:
Bootstrap defaults to `../../contracts` but can be updated to concatenate `call`, per this example.

Reply:
Changed.